Unix/configeditor/configeditor.cpp

```diff
@@ -329,14 +342,18 @@ int MI_MAIN_CALL main(int argc, const char** argv)
 
     // Read from stdin, writing results to stdout with modifications as necessary
 
+    bool fDontEchoFile = opts.query.empty() ? false : true;
```

Comment:
This could have just been `bool fDontEchoFile = !opts.query.empty()`, which is a bit more readable.